### ISSUE TITLE
work release or cancel hangs after restarting control service node

### ISSUE THREAD

**Issue by fosterseth** [ansible/receptor#180]
start `foo.yml` and `bar.yml` nodes
1. on foo, `work submit bar 100cat`
2. let job start, then control + C to kill `foo`
3. bring `foo` back online and attempt a `work release jobID` (or cancel)

it will hang

foo.yaml
```yaml
---
- node:
    id: foo
    allowedpeers: bar
- log-level:
    level: Debug

- control-service:
    service: control
    filename: /tmp/foo.sock

- tcp-listener:
    port: 2222
```

bar.yml
```yaml
---
- node:
    id: bar
    allowedpeers: foo

- tcp-peer:
    address: localhost:2222
    redial: true

- work-command:
    workType: 100cat
    command: bash
    params: "-c \"seq 100 | /home/sfoster/sockceptor/sleepcat.sh\""
```

**Comment by fosterseth** [comment]
found the issue and will put fix into #178

**Comment by squidboylan** [comment]
testing for this was added in https://github.com/project-receptor/receptor/pull/184
